gui/app/components/list_results.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactTable from 'react-table';
+
+const columns = [{
+    Header: 'agencyId',
+    accessor: 'agencyId'
+},{
+    Header: 'bibliographicRecordId',
```

Comment:
muligvis 'Record Id'  er en bedre header i forhold til den meget lange 
tilsvaraende med 'Agency' istedet for cammelCase AgencyId